common/src/main/java/io/apicurio/registry/rest/v2/beans/ArtifactMetaData.java

```diff
@@ -59,7 +59,7 @@
      * (Required)
      * 
      */
-    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss'Z'", timezone = "UTC")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
```

Comment:
Not sure if Z in this scope has any meaning. It is probably ignored.

Reply:
this pattern is generated by apicurio studio, and somehow jackson uses serialize/deserialize our dates
so it's better to keep it like it was

Reply:
Yes. I was just flagging it.